programs/uxd/src/instructions/lsd/borrow_from_lsd_depository.rs

```diff
@@ -0,0 +1,144 @@
+use anchor_lang::prelude::*;
+use anchor_spl::token::Mint;
+use anchor_spl::token::Token;
+use anchor_spl::token::TokenAccount;
+
+use crate::error::UxdError;
+use crate::state::LsdDepository;
+use crate::state::LsdPosition;
+use crate::utils::validate_collateral_amount;
+use crate::utils::validate_loan_to_value_bps;
+use crate::validate_is_program_frozen;
+use crate::Controller;
+use crate::CONTROLLER_NAMESPACE;
+use crate::LSD_DEPOSITORY_NAMESPACE;
+use crate::LSD_POSITION_SPACE;
+use crate::LSD_PROFITS_TOKEN_ACCOUNT_NAMESPACE;
+
+#[derive(Accounts)]
+pub struct BorrowFromLsdDepository<'info> {
+    /// #1
+    pub user: Signer<'info>,
+
+    /// #2
+    #[account(mut)]
+    pub payer: Signer<'info>,
+
+    /// #3
+    #[account(
+        mut,
+        seeds = [CONTROLLER_NAMESPACE],
+        bump = controller.load()?.bump,
+        has_one = redeemable_mint @UxdError::InvalidRedeemableMint
+    )]
+    pub controller: AccountLoader<'info, Controller>,
+
+    /// #4
+    #[account(
+        mut,
+        seeds = [LSD_DEPOSITORY_NAMESPACE, collateral_mint.key().as_ref()],
```

Comment:
should we include the liquidation mint in the depository seeds?

Reply:
That would mean running two LSD depository for the same collateral with different liquidation mints... which we won't do 99.999% chance. Tbh if USDC dies, we die 😓

Reply:
yeah fair enough, I guess my point was: is there any downside to include it? no strong opinion tho